cli/cmd/component.go

```diff
@@ -187,18 +186,8 @@ func (c *cliState) LoadComponents() {
 		}
 
 		version := component.InstalledVersion()
-		componentDir, err := component.Dir()
-		if err != nil {
-			c.Log.Debugw("unable to find component directory", "error", err)
-			return
-		}
-		if devInfo, _ := lwcomponent.NewDevInfo(componentDir); devInfo != nil {
```

Comment:
by removing this `lwcomponent.NewDevInfo ` the function no longer needs to be public?

Reply:
Yep! Updated.